src/main/kotlin/uk/gov/dluhc/printapi/database/entity/PrintRequest.kt

```diff
@@ -93,7 +93,7 @@ class PrintRequest(
     @JoinColumn(name = "print_request_id", nullable = false)
     var statusHistory: MutableList<PrintRequestStatus> = mutableListOf(),
 
-    @CreationTimestamp
```

Comment:
Doesn't populate the reprint Entity that is added to the saved certificate - integration tests fail, so refactored.